docs/src/user_docs/stdlib/main.md

```diff
@@ -10,19 +10,6 @@ The second goal can be achieved because calls to procedures in the standard libr
 ## Organization and usage
 Procedures in Miden standard library are organized into modules, each targeting a narrow set of functionality. Modules are grouped into higher-level namespaces. However, higher-level namespaces do not expose any procedures themselves. For example, `std::math::u64` is a module containing procedures for working with 64-bit unsigned integers. This module is a part of the `std::math` namespace. However, the `std::math` namespace does not expose any procedures.
```

Comment:
in `the Miden standard library`?

Reply:
It is again not super clear to me what the difference is between something that is `organized into` something else compared to something that is `grouped into` something else.

Reply:
I use them as synonyms.